sdk/python/feast/infra/materialization/contrib/spark/spark_materialization_engine.py

```diff
@@ -184,23 +179,13 @@ def _materialize_one(
                     self.repo_config.batch_engine.partitions
                 )
 
-            # Calculate batch_size per feature_view
-            feature_view_batch_size = (
-                self.repo_config.batch_engine.batch_size
-                if "batch_size" not in feature_view.tags
-                or feature_view.tags["batch_size"] is None
-                else int(feature_view.tags["batch_size"])
-            )
-
             print(
```

Comment:
Would be nice to eventually use logging.

Reply:
Python logging doesn't work directly. We need to fix this at some point.